packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -223,7 +224,7 @@ export const RemoteServerFactory = {
      * @private
      */
     _isBundleOrProxyPath(url) {
-        return url.startsWith(proxyBasePath) || url.startsWith(bundleBasePath)
+        return url.includes(proxyBasePath) || url.includes(bundleBasePath)
```

Comment:
Using includes will loosen the condition check here. 🤔  is it possible to keep the `startWith`? Using `includes` allows a lot of variety of url to match Say. `url = /hello-wordl/something-complete-random/mobify/bundle//i-am-sth/testing` will also match /mobidy/bundle => is this acceptable? 
 Can someone mess up this logic and cause bug down the stream?  what if we can catch that by sanitizing the url and ensure the sanitized url with basePath is strictly followed with proxy/path

Reply:
if we know basePath is only one part before url, say /basePath/mobify/bundle. I think we should retain `starWith`  function by sanitizing the url to exclude `basePath`

Reply:
I relaxed the condition here initially because this function is used by the request processor to identify and early exit for any bundle or proxy request on the local environment, regardless of whether it had a base path or not (since we don't want the request processor to run for any bundle or proxy request on local). On MRT, the router in front of Lambda prevents any bundle or proxy requests from reaching the request processor so this code is not executed there. 

But with the `_setupRemoveBasePathFromPathMiddleware` placed before the request processor, the base path should already have been removed by the time the request arrives at the request processor. So I can revert this back to startsWith.